theme/snippets/product-info-bar.liquid

```diff
@@ -45,9 +45,9 @@
       {% endfor %}
     </span>
   </span>
-  <div class="col-12 md:col-5 flex flex-row justify-start md:justify-end px_625 py1 md:px1_875 md:py1_3125 color-white sans-light-md uppercase">
-    <span class="md:none">{{ product.price | money_without_trailing_zeros }}</span>
-    <span class="md:none mx_25">-</span>
+  <div class="col-12 md:col-5 flex flex-row justify-start md:justify-end px_625 py1 md:pr1 md:pl0 md:py1_3125 color-white sans-light-md uppercase">
+    <span class="pt_15 md:none">{{ product.price | money_without_trailing_zeros }}</span>
+    <span class="pt_15 md:none mx_25">-</span>
```

Comment:
Have to add this .15 rem padding top to the Add to Cart button to crop the line height, now that the Add to Cart button has the border on hover. So here I'm adding it to the product price because without it, it looks .15 rem off.